pkgs/top-level/all-packages.nix

```diff
@@ -6081,6 +6081,11 @@ in
 
   hyenae = callPackage ../tools/networking/hyenae { };
 
+  hylafaxp = callPackage ../servers/hylafaxp {
+    pam = null;
+    openldap = null;
```

Comment:
Since ldap and pam aren't useful to me I nulled them out.

But, a more enterprise-centric person might want to look at the module for hfaxd's inetd support that's likely to go with pam. Similarly, ldap...